drivers/gpu/drm/drm_sysctl_freebsd.c

```diff
@@ -90,7 +90,12 @@ drm_sysctl_init(struct drm_device *dev)
 
 	/* Find the next free slot under hw.dri */
 	i = 0;
-	SLIST_FOREACH(oid, SYSCTL_CHILDREN(drioid), oid_link) {
+#if __FreeBSD_version < 1400071
+	SLIST_FOREACH(oid, SYSCTL_CHILDREN(drioid), oid_link)
+#else
+	RB_FOREACH(oid, sysctl_oid_list, SYSCTL_CHILDREN(drioid))
```

Comment:
There is a SYSCTL_FOREACH now, might be better to use this one and simple use ifdef on it rather than the FreeBSD version.

Reply:
Thanks! I force-pushed the modified commit.